packages/es-client/search.js

```diff
@@ -138,9 +138,7 @@ class BaseSearch {
     this.page = Number.parseInt((params.skip) ? params.skip : page, 10);
     this.index = index || defaultIndexAlias;
 
-    if (this.type === process.env.CollectionsTable) {
-      this.hash = 'collectionName';
-    } else if (this.type === process.env.PdrsTable) {
+    if (this.type === process.env.PdrsTable) {
```

Comment:
As noted in the PDR table PR, this seems fine - it's not actually being referenced/used anywhere.